### PR TITLE
Flasher tab: show commit field no matter what, if expert mode enabled

### PR DESCRIPTION
**Pull request by limonspb**
After this PR the "Select commit" text input will show up regardless (if in expert mode)
This will allow triggering cloud build API with make commands specifically for a certain branch/release on top of any PRs.

I think this change also requires Cloud build modification. I can see that request includes # commit:
image
however, the build log does not reflect it:

https://build.betaflight.com/api/builds/24d611898699d3424bbbcdbc2a837ad3/log
https://build.betaflight.com/api/builds/24d611898699d3424bbbcdbc2a837ad3/json

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by limonspb** [comment]
@haslinghuis don't merge plz, till @blckmn have a look. I think it requires some cloud-build changes.

**Comment by blckmn** [comment]
I don't support this change. The commit dialogue is available for zulu "pre"-releases. Releases should not be modifiable.

**Review by blckmn — CHANGES_REQUESTED** [review]

**Comment by blckmn** [comment]
We add a 4.4.3-zulu and after 4.4.3 release we add a 4.4.4-zulu and withdraw the 4.4.3-zulu

**Comment by limonspb** [comment]
> We add a 4.4.3-zulu and after 4.4.3 release we add a 4.4.4-zulu and withdraw the 4.4.3-zulu



> We add a 4.4.3-zulu and after 4.4.3 release we add a 4.4.4-zulu and withdraw the 4.4.3-zulu
> 
> These are pinned to 4.4-maintenance like 4.5.0-zulu is currently pinned to master.

should we just always have 4.4-zulu so you don't have to manually change/add 4.4.2-zulu to 4.4.3-zulu etc?
For example, i wanted to test (https://github.com/betaflight/betaflight/pull/12866) but can't do it now because it's raised for 4.4-maintenance. So i tested the original PR against master, but the code is a bit different in master obviously.

**Pull request closed without merging**
